core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpScreenItem.kt

```diff
@@ -0,0 +1,125 @@
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun HelpScreenItem(
+  modifier: Modifier = Modifier,
+  data: HelpScreenItemDataClass,
+  initiallyOpened: Boolean = false
+) {
+  var isOpen by remember { mutableStateOf(initiallyOpened) }
+  val isDarkTheme = isSystemInDarkTheme()
+  val itemColor = if (isDarkTheme) Color.White else Color.Black
+  val arrowRotation by animateFloatAsState(
+    targetValue = if (isOpen) 180f else 0f,
+    animationSpec = tween(300),
+    label = "arrowRotation"
+  )
+
+  val interactionSource = remember(::MutableInteractionSource)
+
+  Column(
+    modifier = modifier
+      .fillMaxWidth()
+      .padding(top = 12.dp),
+    verticalArrangement = Arrangement.Center,
+    horizontalAlignment = Alignment.CenterHorizontally
+  ) {
+    Row(
+      horizontalArrangement = Arrangement.SpaceBetween,
+      verticalAlignment = Alignment.CenterVertically,
+      modifier = Modifier
+        .fillMaxWidth()
+        .clickable(interactionSource = interactionSource, indication = null, onClick = {
+          isOpen = !isOpen
+        })
+        .padding(horizontal = 16.dp)
+    ) {
+      Text(
+        text = data.title,
+        fontSize = 18.sp,
+        color = itemColor,
+        fontWeight = FontWeight.SemiBold
+      )
+      Icon(
+        imageVector = Icons.Default.KeyboardArrowDown,
+        contentDescription = "Open or Close DropDown",
+        modifier = Modifier
+          .graphicsLayer {
+            rotationZ = arrowRotation
+          }
+          .size(46.dp),
+        tint = itemColor
+      )
+    }
+
+    Spacer(modifier = Modifier.height(12.dp))
+
+    AnimatedVisibility(visible = isOpen) {
+      Box(
+        contentAlignment = Alignment.Center,
+        modifier = Modifier
+          .fillMaxWidth()
+          .padding(start = 16.dp, end = 16.dp)
+      ) {
+        Text(
+          text = data.description,
```

Comment:
The internal text is showing bigger it and looks different from the previous one. Also, the `side image` for opening and closing the menu items is bigger which does not look good. See the difference:

| Previous design  | Current design |
| ------------- | ------------- |
| ![PreviousDesign](https://github.com/user-attachments/assets/ca03144e-49cd-48d6-9621-12fdd94b32b9) |  ![CurrentDesign](https://github.com/user-attachments/assets/078d730f-3bdb-465e-b41b-3a83c0052a71) |


You can take the reference from xml design how we have used sizes, margins, paddings, etc, for text and images.